lib/store/ledger.ex

```diff
@@ -9,12 +9,85 @@ defmodule Elixium.Store.Ledger do
 
   @store_dir ".chaindata"
   @ets_name :chaindata
+  @block_cache :block_cache
```

Comment:
The `:chaindata` ets table takes care of all caching for blocks. This issue tackles a problem with block orphaning, lets revisit this functionality updating the orphan pool module instead.